web/docs/tql2/operators/http.md

```diff
@@ -0,0 +1,218 @@
+# http
+
+Sends HTTP/1.1 requests and forwards the response.
+
+```tql
+http url:string, [method=string, payload=string, headers=record,
+    response_field=field, metadata_field=field, paginate=string,
+    paginate_delay=duration, parallel=int, tls=bool, certfile=string,
+    keyfile=string, password=string, connection_timeout=duration,
+    max_retry_count=int, retry_delay=duration] { … }
+```
+
+## Description
+
+The `http` operator issues HTTP/1.1 requests and forwards received responses as events.
+
+### `url: string`
+
+URL to connect to.
+
+### `method = string (optional)`
+
+One of the following HTTP method to use when using the client:
+- `get`
+- `head`
+- `post`
+- `put`
+- `del`
+- `connect`
+- `options`
+- `trace`
+
+Defaults to `get`, or `post` if `payload` is specified.
+
+### `payload = string (optional)`
+
+Payload to send with the HTTP request.
+
+### `headers = record (optional)`
+
+Record of headers to send with the request.
+
+### `response_field = field (optional)`
+
+Field to insert the response into. 
+
+Defaults to `this`.
+
+### `metadata_field = field (optional)`
+
+Field to insert metadata into when using the parsing pipeline.
+
+The metadata has the following schema:
+
+| Field                | Type     | Description                           |
+| :------------------- | :------- | :------------------------------------ |
+| `code`               | `uint64` | The HTTP status code of the response. |
+| `headers`            | `record` | The response headers.                 |
+
+### `paginate = string (optional)`
+
+An expression to evaluate against the result of the request (optionally parsed
+by the given pipeline). If the expression evaluation is successful and non-null, the
+resulting string is used as the URL for a new GET request with the same headers.
+
+### `paginate_delay = duration (optional)`
+
+The duration to wait between consecutive paginatation requests.
+
+Defaults to `0s`.
+
+### `parallel = int (optional)`
+
+Maximum amount of requests that can be in progress at any time.
+
+Defaults to `1`.
+
+### `tls = bool (optional)`
+
+Enables TLS.
+
+### `certfile = string (optional)`
+
+Path to the client certificate.
+
+### `keyfile = string (optional)`
+
+Path to the key for the client certificate.
+
+### `password = string (optional)`
+
+Password for keyfile.
+
+### `connection_timeout = duration (optional)`
+
+Timeout for the connection.
+
+Defaults to `5s`.
+
+### `max_retry_count = int (optional)`
+
+The maximum times to retry a failed request. Every request has its own retry
+count.
+
+Defaults to `0`.
+
+### `retry_delay = duration (optional)`
+
+Duration to wait between each retry.
```

Comment:
```suggestion
A duration to wait between each retry.
```
I think you should add articles for all the option names in their descriptions. Omitting them sounds weird to me.